app/src/main/java/com/byeboo/app/presentation/quest/component/QuestCardContent.kt

```
@@ -0,0 +1,137 @@
+) {
+    QuestCardContainer(
+        modifier = modifier,
+        backgroundColor = ByeBooTheme.colors.whiteAlpha10,
+        borderColor = ByeBooTheme.colors.secondary300
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = String.format("%02d", questNumber),
```

detekt (1.23.8)

[warning] String.format("%02d", questNumber) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)